__test__/__snapshots__/regression.test.js.snap

```diff
@@ -337,30 +337,31 @@ false
 [32m[debug.log] done[0m
 [32m[debug.log] ==== TreeSet ====[0m
 [32m[debug.log] [0m
-[32m[debug.log] 1,2,3[0m
+[32m[debug.log] 2,1,3[0m
 [32m[debug.log] check rotations:[0m
-[32m[debug.log] 1,2,3[0m
-[32m[debug.log] 1,2,3[0m
-[32m[debug.log] 1,2,3[0m
-[32m[debug.log] 1,2,3[0m
-[32m[debug.log] 4,8,12,16,20,24,28[0m
-[32m[debug.log] -1,0,1,2,3,4,8,12,16,20,24,28[0m
-[32m[debug.log] 4,8,12,16,20,24,28,29,30,31,32,33[0m
-[32m[debug.log] 4,8,9,10,11,12,13,15,16,20,24,28[0m
-[32m[debug.log] 4,8,12,16,17,19,20,21,22,23,24,28[0m
+[32m[debug.log] 2,1,3[0m
+[32m[debug.log] 2,1,3[0m
+[32m[debug.log] 2,1,3[0m
+[32m[debug.log] 2,1,3[0m
+[32m[debug.log] 16,8,24,4,12,20,28[0m
+[32m[debug.log] 3,1,16,0,2,8,24,-1,4,12,20,28[0m
+[32m[debug.log] 29,16,31,8,24,30,32,4,12,20,28,33[0m
+[32m[debug.log] 12,9,16,8,10,15,24,4,11,20,28[0m
+[32m[debug.log] 20,16,23,8,17,22,24,4,12,19,28[0m
```

Comment:
I manually checked the correctness of these results using https://www.cs.usfca.edu/~galles/visualization/AVLtree.html